### DISCUSSION TITLE
unable to install epg dependencies

### DISCUSSION THREAD

**Discussion by Asif3850** [iptv-org/community#1009] [Q&A]
getting following while installing dependencies in windows 10 pc

"The filename, directory name, or volume label syntax is incorrect.
npm ERR! code 1"

**Comment by freearhey** [comment]
According to this thread:

> this only happens when you use extended characters in a name (anything outside a-z A-Z or 0-9).

and

> The solution is to find and edit it to the correct path.

https://forums.tomshardware.com/threads/file-name-directory-name-or-volume-label-syntax-is-incorrect.1708093/
